LiskHQ/lisk-sdk
# Fix ABI schema to align with interface - Closes #7355

### What was the problem?

This PR resolves #7355 

### How was it solved?

Remove unnecessary schema definition

### How was it tested?

- Remove data path `~/.lisk/dpos-mainchain`
- Run dpos example and observe it runs
